scripts/flitcli/flit_bisect.py

```diff
@@ -922,7 +869,7 @@ def bisect_search(score_func, elements, found_callback=None,
             .format(differing_element)
         differing_list.append((differing_element, score))
         # inform caller that a differing element was found
-        if found_callback != None:
+        if found_callback is not None:
```

Comment:
Thank you Ian.  I am fully aware how `is` is different from `==`.  It is the preferred way to check equality for `None` because you may have a bug in your code for the `_compare_()` method (I'm not exactly certain about the name of the method) where the object will be deemed equal to `None` by mistake.

Thanks for the little demonstration :smile: I only use `is` for instance equality, which is almost always for `None`.